week-5/D-webchat/exercise-3.js

```diff
@@ -42,4 +44,40 @@ Task 4
 Use the setInterval function to reload automatically the messages of your webchat every 2 seconds.
 The code responsible to show the messages in the page is in exercise-1.js, so you will need to write your code there :-)
 */
-
+let ticks = 0;
+var requestURL = "https://codeyourfuture.herokuapp.com/api/messages";
+
+var xhttp = new XMLHttpRequest();
+xhttp.onreadystatechange = function() {
```

Comment:
You don't need to use `XMLHttpRequest` here if you are using `fetch`. This code will actually send 2 requests to the same URL because you are using both. I would stick to `fetch` as not many developers use `XMLHttpRequest` any more.

Reply:
I'll remove it.